digitalocean/resource_digitalocean_container_registry.go

```diff
@@ -0,0 +1,84 @@
+package digitalocean
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/digitalocean/godo"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+)
+
+func resourceDigitalOceanContainerRegistry() *schema.Resource {
+	return &schema.Resource{
```

Comment:
Do we have other resources without an update?

Reply:
Tag doesn't have an update. Also currently in the beta you can't change the name of the registry so there is nothing to update.